docs/tutorials/04_encrypted_ebs.rst

```diff
@@ -3,12 +3,12 @@
 .. toctree::
    :maxdepth: 2
 
-###################################
-Encrypted EBS with a Custom KMS Key
-###################################
+#####################################
+Disk Encryption with a Custom KMS Key
```

Comment:
Shall we modify the filename as well?

Reply:
Modified `04_encrypted_ebs.rst -> 04_encrypted_kms_fs.rst`